README.md

```diff
@@ -39,3 +39,22 @@ const bucket2 = new Bucket(this, 'AnotherImageryArchive');
 applyTags(bucket1, commonTags);
 applyTags(bucket2, commonTags);
 ```
+## Contributing
+
+### Adding responder teams.
+
+Contact the prod ops team in slack at [#team-step-prod-ops](https://linz.enterprise.slack.com/archives/C05Q11EGLA0) 
+To create a new team in [opsgenie](https://toitutewhenua.app.opsgenie.com/teams/list)
+once this team is created, add a new responder team to `src/responder-teams.ts` file with the new team name. These two names 
+must match to ensure New Relic alerts are sent to the correct team.
+
+### Renaming responder teams.
+
+When renaming a responder team, contact the prod ops team in slack at [#team-step-prod-ops](https://linz.enterprise.slack.com/archives/C05Q11EGLA0)
+once the name as been updated, update `src/responder-teams.ts` to match the new name. And ensure you update the `responderTeam` in your `applyTags()` function.
+
+### Removing responder teams.
+
+When removing a responder team, contact the prod ops team in slack at [#team-step-prod-ops](https://linz.enterprise.slack.com/archives/C05Q11EGLA0)
+once the team has been removed, remove the team from `src/responder-teams.ts` file.
```

Comment:
Same thing in this case, can we please mention that only ProdOps will remove or amend Teams in Opsgenie?